spec/models/endpoint_spec.rb

```diff
@@ -15,13 +15,23 @@
       endpoint.save!
     end
 
-    it "deltes cloudformation stack" do
+    it "deletes cloudformation stack" do
       endpoint.save!
       expect_any_instance_of(CloudFormation::Executor).to receive(:delete)
       endpoint.destroy!
     end
   end
 
+  describe "#validations" do
```

Comment:
`shoulda-matchers` doesn't support scope with association.
so I wrote the code like this.
https://github.com/thoughtbot/shoulda-matchers/issues/814